swupd/manifest_test.go

```diff
@@ -275,19 +275,15 @@ func TestWriteManifestFile(t *testing.T) {
 		t.Fatal("ReadManifestFromFile did not add file entried to the file list")
 	}
 
-	f, err := ioutil.TempFile("testdata", "manifest.result")
-	if err != nil {
-		t.Fatal("unable to open file for write")
-	}
-	defer os.Remove(f.Name())
-
-	newpath := f.Name()
+	// do not use a tempfile here, we just need the unique name
```

Comment:
Minor: this is a change in the wrong direction IMHO. If this test is copy/pasted to create a new one (a reasonable way to create a new test), we will have a problem if they run in parallel.

Reply:
Maybe, but all we need in that case is a unique name. The function takes a pathname, not a file, so creating a file here is not necessary. Is it really important to protect against future copy-paste issues now?